interface/patient_file/summary/demographics.php

```diff
@@ -1486,6 +1486,45 @@ function setMyPatient() {
       } // End of Appointments.
 
 
+      /* Widget that shows recurrences for appointments. */
+     if (isset($pid) && !$GLOBALS['disable_calendar'] && $GLOBALS['recurrences_widget']) {
+
+         $widgetTitle = xl("Appointment Recurrences");
+         $widgetLabel = "appointment_recurrences";
+         $widgetButtonLabel = xl("Add");
+         $widgetButtonLink = "return newEvt();";
+         $widgetButtonClass = "";
+         $linkMethod = "javascript";
+         $bodyClass = "summary_item small";
+         $widgetAuth = false;
+         $fixedWidth = false;
+         expand_collapse_widget($widgetTitle, $widgetLabel, $widgetButtonLabel, $widgetButtonLink, $widgetButtonClass, $linkMethod, $bodyClass, $widgetAuth, $fixedWidth);
+         $count = 0;
+         $toggleSet = true;
+         $priorDate = "";
+
+         //Fetch patient's recurrences. Function returns array with recurrence appointments' category, recurrence pattern (interpreted), and end date.
+         $recurrences = fetchRecurrences($pid);
+
+         foreach ($recurrences as $row) {
+             //checks if there are recurrences and if they are current (git didn't end yet)
+             if($row == false || !recurrence_is_current($row['pc_endDate']))
+                 continue;
+             echo "<div>";
+             echo "<span>" . xlt('Appointment Category: ') . htmlspecialchars($row['pc_title'], ENT_NOQUOTES) . "</span>";
```

Comment:
translation constants should not contain leading spaces or colons:
``````
xlt('Appointment Category') . ': '